bigtable/google/cloud/bigtable/row.py

```diff
@@ -582,6 +582,7 @@ def commit(self):
             table_name=self._table.name,
             row_key=self._row_key,
             predicate_filter=self._filter.to_pb(),
+            app_profile_id=self._table._app_profile_id,
```

Comment:
Thanks for adding this! Would you be able to add a unit test for this as well?

Reply:
hey @kolea2 see 20a4838f85689399b8ebfeb0ed152c08733d2010, let me know if there is a better way to access the arguments passed to the mock